components/Navigation.js

```diff
@@ -1,38 +1,80 @@
 import Link from "next/link";
 import styled from "styled-components";
-import IconAdd from "@/public/iconAdd.svg";
-import IconHome from "@/public/iconHome.svg";
+import AddIcon from "@/public/icons/plus.svg";
+import HomeIcon from "@/public/icons/home.svg";
 import { useAtom } from "jotai";
 import { statusUploadAtom } from "@/lib/atom";
+import { useRouter } from "next/router";
+import { useState } from "react";
 
 export default function Navigation() {
   const [statusUpload, setStatusUpload] = useAtom(statusUploadAtom);
+  const router = useRouter();
 
   return (
-    <List>
-      <li>
-        <Link href="/" onClick={() => setStatusUpload("")}>
-          <IconHome />
-        </Link>
-      </li>
-      <li>
-        <Link href="/create-new-project">
-          <IconAdd />
-        </Link>
-      </li>
-    </List>
+    <StyledNavigation>
+      <List>
+        <li>
+          <Link href="/" onClick={() => setStatusUpload("")}>
+            <HomeIconWrapper isFocused={router.pathname}>
+              <HomeIcon />
+            </HomeIconWrapper>
+          </Link>
+        </li>
+        <li>
+          <Link href="/create-new-project">
+            <AddIconWrapper isFocused={router.pathname}>
+              <AddIcon />
+            </AddIconWrapper>
+          </Link>
+        </li>
+      </List>
+    </StyledNavigation>
   );
 }
 
+const HomeIconWrapper = styled.div`
+  display: flex;
+  width: 2.8em;
+  border-radius: 50%;
+  background-color: ${({ isFocused }) =>
+    isFocused === "/"
+      ? "var( --color-background)"
+      : "var(--color-buttons-yellow)"};
+  &:active {
+    position: relative;
+    top: 1px;
+  }
+`;
+
+const AddIconWrapper = styled.div`
+  display: flex;
+  width: 2.8em;
+  border-radius: 50%;
+  background-color: ${({ isFocused }) =>
+    isFocused === "/create-new-project"
+      ? "var( --color-background)"
+      : "var(--color-buttons-yellow)"};
+  &:active {
+    position: relative;
+    top: 1px;
+  }
+`;
```

Comment:
I would also recommend to work with one IconWrapper. See the art gallery example above to reduce your code. :)